File: Week 4/perfectRect/hyewon.cpp
#include <math.h>

using namespace std;

long long solution(int w, int h) {
	long long answer = 0;

	for (int i = 1; i < w; i++) {
		answer += floor((double)-h / w * i + h); //³»¸²
	}
	answer *= 2; //for rectangle
	return answer;
}

File: Week11/Camouflage/hyewon.py
def solution(clothes):
    answer = 1
    dic = {}
    for cloth in clothes:
        if cloth[1] not in dic:
            dic[cloth[1]] = 1
        else:
            dic[cloth[1]] += 1

    for val in dic.values():
        answer *= (val + 1)
    answer -= 1

    return answer

print(solution([["yellow_hat", "headgear"], ["blue_sunglasses", "eyewear"], ["green_turban", "headgear"]]))
